### ISSUE TITLE
Irrelevant button label in the transcribe panel

### ISSUE THREAD

**Issue by cyberbaloo** [suitenumerique/meet#873]
## Bug Report

**Problematic behavior**
The button to return to "meeting tools" does not have an accessible label
The left arrow is a decorative image.

**Expected behavior/code**
Add a "Back to meeting tools" label to the button
Add a text alternative `alt=""` to the image.
The image must be ignored and the button must be rendered.

**Additional context/Screenshots**
<img width="389" height="813" alt="" src="https://github.com/user-attachments/assets/ab53f7bb-79f6-471d-ae75-3e19db72f4bf" />
